### DISCUSSION TITLE
HTTP-Fehler: 500 Internal Server Error - Paperless

### DISCUSSION THREAD

**Discussion by MrNoooooooob** [jonaswinkler/paperless-ng#869] [Support]
Problems with uploading documents in PDF an error always occurs Raspberry Pi - Portainer Installed

<img width="577" alt="Bildschirmfoto 2021-04-05 um 20 26 12" src="https://user-images.githubusercontent.com/70014906/113610327-3d4c7080-964d-11eb-8306-077e48e6cb4b.png">
[_paperless_web_logs-2.txt](https://github.com/jonaswinkler/paperless-ng/files/6259819/_paperless_web_logs-2.txt)
<img width="400" alt="Bildschirmfoto 2021-04-05 um 20 08 16" src="https://user-images.githubusercontent.com/70014906/113609100-a0d59e80-964b-11eb-88b6-9eb6dd689674.png">
<img width="1412" alt="Bildschirmfoto 2021-04-05 um 20 15 58" src="https://user-images.githubusercontent.com/70014906/113609215-c662a800-964b-11eb-8268-07b0c9dd46d8.png">

Thank You for Help

**Comment by jonaswinkler** [comment]
Please use the docker-consume files provided in the repository and make modifications to these as required.

* Your connection string for redis is wrong.
* Don't execute runserver directly.
* You don't need to have a separate container for the consumer anymore.
